internal/services/storage/storage_object_replication_resource.go

```diff
@@ -66,14 +66,12 @@ func resourceStorageObjectReplication() *pluginsdk.Resource {
 						"source_container_name": {
 							Type:         pluginsdk.TypeString,
 							Required:     true,
-							ForceNew:     true,
```

Comment:
These ForceNews aren't needed anymore. I checked to make sure that it didn't cause issues with this resource by updating them to different things and Azure accepted them just fine